db/migrate/20240320185213_create_collections.rb

```diff
@@ -1,5 +1,7 @@
 class CreateCollections < ActiveRecord::Migration[7.1]
   def change
+    return if table_exists?(:collections)
```

Comment:
Does this file need to change? I think this breaks `db:rollback`, and the file move would put those who have already migrated into a strange state where a migration file is missing.